db/schema.rb

```diff
@@ -225,6 +225,16 @@
     t.index ["location_id"], name: "index_ndcs_on_location_id"
   end
 
+  create_table "wb_extra_country_data", force: :cascade do |t|
```

Comment:
please run the changes you made to the migrations so that you can commit an up to date version of schema.rb (this version seems outdated)